### PR TITLE
[VL] Add velox nth_value function support

### PR DESCRIPTION
**Pull request by zhli1142015**
## What changes were proposed in this pull request?

Add velox nth_value function support.

## How was this patch tested?

UT.

### CODE REVIEW

**Comment by zhli1142015** [comment]
@JkSelf  and @rui-mo , could you please take a look?
Thanks.

File: ep/build-velox/src/get_velox.sh [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/zhli1142015/velox.git
+VELOX_BRANCH=window-refactor
```
Comment (rui-mo) [inline ep/build-velox/src/get_velox.sh:6]: The Velox PR is merged, so please revert this change. Thanks.

**Review by rui-mo — DISMISSED** [review]

**Review by rui-mo — APPROVED** [review]
